code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm

```diff
@@ -0,0 +1,377 @@
+
+	virus_suspectibility = 2
+	resulting_atoms = list(/mob/living/simple_animal/hostile/carp = 1)
+
+/datum/micro_organism/cell_line/megacarp
+	desc = "Cartilaginous cyprinid cells"
+	required_reagents = list(
+						/datum/reagent/consumable/nutriment/protein,
+						/datum/reagent/medicine/C2/instabitaluri,
+						/datum/reagent/consumable/nutriment)
+
+	supplementary_reagents = list(
+						/datum/reagent/consumable/cornoil = 4,
+						/datum/reagent/growthserum = 3,
+						/datum/reagent/toxin/carpotoxin = 2,
+						/datum/reagent/consumable/cooking_oil = 2,
+						/datum/reagent/consumable/nutriment/vitamin = 2)
+
+	suppressive_reagents = list(
+						/datum/reagent/toxin/bungotoxin = -6,
+						/datum/reagent/oxygen = -3)
+
+	virus_suspectibility = 1
+	resulting_atoms = list(/mob/living/simple_animal/hostile/carp/megacarp = 1)
+
+/datum/micro_organism/cell_line/snake
+	desc = "Ophidic cells"
+	required_reagents = list(
+						/datum/reagent/consumable/nutriment/protein,
+						/datum/reagent/liquidgibs)
+
+	supplementary_reagents = list(
+						/datum/reagent/growthserum = 3,
+						/datum/reagent/consumable/nutriment/peptides = 3,
+						/datum/reagent/consumable/eggyolk = 2,
+						/datum/reagent/consumable/nutriment/vitamin = 2)
+
+	suppressive_reagents = list(
+						/datum/reagent/consumable/corn_syrup = -6,
+						/datum/reagent/sulfur = -3) //sulfur repels snakes according to professor google.
+
+	resulting_atoms = list(/mob/living/simple_animal/hostile/retaliate/poison/snake = 1)
+
+
+///////////////////////////////////////////
+/// 		SLIMES, OOZES & BLOBS  		///
+//////////////////////////////////////////
+
+/datum/micro_organism/cell_line/slime
+	desc = "Slime particles"
+	required_reagents = list(/datum/reagent/consumable/nutriment/protein)
+
+	supplementary_reagents = list(
+	 					/datum/reagent/toxin/slimejelly = 2,
+	 					/datum/reagent/liquidgibs = 2,
+						/datum/reagent/consumable/enzyme = 1)
+
+	suppressive_reagents = list(
+						/datum/reagent/consumable/frostoil = -4,
+						/datum/reagent/cryostylane = -4,
+						/datum/reagent/medicine/morphine = -2,
+						/datum/reagent/consumable/ice = -2) //Brrr!
+
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/slime = 1)
+
+/datum/micro_organism/cell_line/blob_spore //shitty cell line to dilute the pool, feel free to make easier to grow if it doesn't interfer with growing the powerful mobs enough.
+	desc = "Immature blob spores"
+	required_reagents = list(/datum/reagent/consumable/nutriment/protein)
+
+	supplementary_reagents = list(
+	 					/datum/reagent/consumable/nutriment/vitamin = 3,
+	 					/datum/reagent/liquidgibs = 2,
+	 					/datum/reagent/sulfur = 2)
+
+	suppressive_reagents = list(
+						/datum/reagent/consumable/tinlux = -6,
+						/datum/reagent/napalm = -4)
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/hostile/blob/blobspore = 2) //These are useless so we might as well spawn 2.
```

Comment:
Actually, with the recent blob changes, blob spores are a decent mob to have.

You can give a blob spore a corpse to infect, and a ghost can click on the resulting blob zombie to possess it.

I personally don't think that this is a problem, but hey, I just thought that you should be made aware that blob spores aren't "useless".